server_test.go

```diff
@@ -3413,8 +3425,8 @@ func TestMaxBodySizePerRequest(t *testing.T) {
 func TestStreamRequestBody(t *testing.T) {
 	t.Parallel()
 
-	part1 := strings.Repeat("1", 1<<10)
-	part2 := strings.Repeat("2", 1<<20-1<<10)
+	part1 := strings.Repeat("1", 1<<15)
```

Comment:
Had to increase this size. If the body is big enough we always block on chunks of 8kb. So when `part1` is smaller than 8kb we get a read timeout.